content/patterns/disclosure/examples/disclosure-image-description.html

```diff
@@ -245,61 +244,55 @@ <h3 id="id_data_label">Data for Charles Minard's Chart of Napoleon's Invasion of
           </li>
           <li>
-            To ensure the inline SVG arrow graphics in the CSS have sufficient contrast with the background when high contrast settings invert colors, the color of the arrows are synchronized with the color of the text content.
-            For example, the color of the arrow is set to match the foreground color of high contrast mode text by specifying the CSS <code>currentcolor</code> value for the <code>stroke</code> and <code>fill</code> properties of the <code>polygon</code> elements used to draw the arrows.
-            If specific colors were instead used to specify the <code>polygon</code> properties, those colors would remain the same in high contrast mode, which could lead to insufficient contrast between the arrows and the background or even make the arrows invisible if the color matched the high contrast mode background.<br>
-            Note: The SVG element needs to have the CSS <code>forced-color-adjust</code> property set to <code>auto</code> for the <code>currentcolor</code> value to be updated in high contrast mode.
-            Some browsers do not use <code>auto</code> for the default value.
+            ハイコントラスト設定で色が反転したときに CSS 内のインライン SVG 矢印グラフィックが背景と十分なコントラストを持つように、矢印の色はテキストコンテンツの色と同期されます。たとえば、矢印を描画するために使われる <code>polygon</code> 要素の <code>stroke</code> および <code>fill</code> プロパティに CSS の <code>currentcolor</code> 値を指定することにより、ハイコントラストモードテキストの前景色に合わせて矢印の色が設定されます。<code>polygon</code> プロパティに特定の色を使用してしまうと、それらの色はハイコントラストモードで同じままになり、矢印と背景の間のコントラストが不十分になるか、色がハイコントラストモードの背景と一致する場合は矢印が見えなくなる可能性があります。<br>
+            注記：SVG 要素は、ハイコントラストモードで <code>currentcolor</code> 値が更新されるように、CSS の <code>forced-color-adjust</code> プロパティを <code>auto</code> に設定する必要があります。一部のブラウザーはデフォルト値として <code>auto</code> を使用しません。
           </li>
         </ul>
       </section>
 
       <section>
-        <h2 id="kbd_label">Keyboard Support</h2>
+        <h2 id="kbd_label">キーボードサポート</h2>
         <table aria-labelledby="kbd_label" class="def">
           <thead>
             <tr>
-              <th>Key</th>
-              <th>Function</th>
+              <th>キー</th>
+              <th>機能</th>
             </tr>
           </thead>
           <tbody>
             <tr data-test-id="key-tab">
               <th>
                 <kbd>Tab</kbd>
               </th>
-              <td>Moves keyboard focus to the disclosure button.</td>
+              <td>キーボードフォーカスをディスクロージャーボタンに移動する。</td>
             </tr>
             <tr data-test-id="key-space-or-enter">
               <th>
-                <kbd>Space</kbd> or <br>
+                <kbd>Space</kbd> 又は<br>
                 <kbd>Enter</kbd>
               </th>
-              <td>Activates the disclosure button, which toggles the visibility of the long description.</td>
+              <td>ディスクロージャーボタンをアクティベートし、長い説明の表示を切り替えます。</td>
```

Comment:
意訳気味になりますが、若干手を加えてみました。

```suggestion
              <td>ディスクロージャーボタンをアクティベートし、長い説明を表示するかどうかを切り替えます。</td>
```